examples/replay_with_viewer.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+Simple trajectory replay script using teleop wrapper.
+"""
+
+import sys
+import os
+import genesis as gs
+
+# Add the src directory to the path so we can import our modules
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'src'))
+
+from agent.gs_agent.wrappers.teleop_wrapper import TeleopWrapper
+from env.gs_env.sim.envs.so101_cube_env import SO101CubeEnv
+
```

Comment:
It is not the correct way of importing the python packages. It is recommended to use `uv` to first install `gs_env` via `uv sync --package gs-env`. Afterwards, you should be able to import `gs_agent` or `gs_env` directly without `sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'src'))`

```python
from gs_agent.wrappers.teleop_wrapper import TeleopWrapper
from gs_env.sim.envs.so101_cube_env import SO101CubeEnv
```